src/rebar3.erl

```diff
@@ -76,7 +76,8 @@ main(Args) ->
 run(BaseState, Commands) ->
     _ = application:load(rebar),
     BaseState1 = rebar_state:set(BaseState, task, Commands),
-    run_aux(BaseState1, [], Commands).
+    BaseState2 = rebar_state:set(BaseState1, caller, api),
```

Comment:
Is the issue here running as a shell or as a program?

I.e. is it possible that anyone calling rebar from a custom shell program will see a failure? Do we care?